kubecargoload.py

```diff
@@ -6,7 +6,7 @@
 
 """
 List PODs of a specific namespace or all namespaces with their
-configured memory or cpu requests, limits and the current memory or cpu usage.
+configured memory requests, limits and the current memory usage.
```

Comment:
Did you remove the "cpu" part here on purpose?

Reply:
By mistake.